backend/connector-integration/src/connectors/globalpay/transformers.rs

```diff
@@ -276,7 +277,7 @@ pub struct Initiator {
     pub stored_credential: Option<String>,
 }
 
```

Comment:
Similarly, `Clone` and `Copy` traits are added but their necessity is unclear.

```suggestion
#[derive(Debug, Serialize)]
```